pioneers-testnet/pioneers-exercise-3.md

```diff
@@ -31,7 +31,14 @@ cabal build all
 cp …/cardano-node ~/.local/bin
 cp …/cardano-cli ~/.local/bin
 ```
-Note that the address format has changed on the Pioneer testnet, so you will need to generate a new address, and request some Ada.
+Note that the address format has changed on the Pioneer testnet, so you will need to generate a new address, and [request](https://docs.google.com/spreadsheets/d/1o62S2_24bcZNIpT3FASKHVGHG-dQ4J0vDaVCEsxbVKU/edit?usp=sharing) some Ada.
+
+```
+cardano-cli shelley address key-gen …
+cardano-cli shelley address build …
+```
+
+You'll also need to get matching configuration files from <https://hydra.iohk.io/build/2674253/download/1/index.html>
```

Comment:
Not sure about the link. It works today, but will get stale with next update - the only link I've been able to find is even older and not working with pioneer-3, in https://github.com/input-output-hk/cardano-tutorials/blob/master/node-setup/GuideForExcercise1.md